spec/acceptance/foreman_cli_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper_acceptance'
+
+describe 'Scenario: install foreman-cli without foreman' do
+  before(:context) do
```

Comment:
My thinking was that this ensures a clean test. Beaker can reuse the container for multiple tests and adding cleanup in every test is likely to miss something.